test/unit/Fixture/StaticPropertyGetSet.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace BetterReflection\Fixture\StaticPropertyGetSet;
+
+class Foo
+{
+}
+
+class Bar
+{
+    public static $baz;
```

Comment:
Test also with protected and private (special care about private + parent class private with same name)